src/main/java/com/spotinst/sdkjava/model/api/azure/statefulNode/ApiLaunchSpecNetworkInterfacesConfiguration.java

```diff
@@ -0,0 +1,110 @@
+package com.spotinst.sdkjava.model.api.azure.statefulNode;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiLaunchSpecNetworkInterfacesConfiguration implements IPartialUpdateEntity {
```

Comment:
missing multiple parameters such as applicationSecurityGroup, publicIps array of objects,